src/update/balrog.cpp

```diff
@@ -461,3 +485,34 @@ void Balrog::propagateError(NetworkRequest* request,
 
   REPORTNETWORKERROR(error, m_errorPropagationPolicy, "balrog");
 }
+
+// static
+QString Balrog::balrogUrl() {
+  const QString product = "FirefoxVPN";
+  QString hostname = Constants::BALROG_PROD_HOSTNAME;
+  QString channel = "release";
+
+  if (Feature::get(Feature::Feature_stagingUpdateServer)->isSupported()) {
```

Comment:
Does this allow for runtime switching to stage? If not, is that something you want to do? (It would probably make future testing easier not to require special builds...)

Reply:
Yes, this feature can be switched at runtime through the developer menu.